### ISSUE TITLE
Permission Exception Guard for clpipe.log

### ISSUE THREAD

**Issue by wasciutto** [cohenlabUNC/clpipe#296]
Due to how common it is for users to not have the right permissions set, if the clpipe.log file is inaccessible for a particular user for a particular project, this should be caught and handled. clpipe.log not being accessible makes all of clpipe not usable. Although we are implementing group write permission to this file by default (#297), the file could have been created in an older version of clpipe, or the permissions could get changed accidentally. 

This should be handled here: https://github.com/cohenlabUNC/clpipe/blob/05ef6d404a6407ab5d6ce961c4b68b43d2719261/clpipe/utils.py#L74

Basically, find a way to test that the user has write access to the "log_file", and if they don't, exit and don't add the file handler to the logger. Function should log a warning with the console logger passed in.

I think to test this, you can make a clpipe.log file, and remove your own write permissions from it.

**Comment by wasciutto** [comment]
closed by #315
